Products/CMFPlone/resources/browser/scripts.pt

```diff
@@ -10,16 +10,14 @@
 </tal:if><tal:if condition="condcomment">
     <tal:opencc tal:replace="structure string:&lt;!--[if ${condcomment}]&gt;" />
 </tal:if>
-  <script type="text/javascript"
-      tal:attributes="
-        src script/src;
-        data-bundle script/bundle;
-        async script/async|nothing;
-        defer script/defer|nothing"/>
+  <script tal:attributes="src script/src;
+                          data-bundle script/bundle;
+                          async script/async|nothing;
+                          defer script/defer|nothing"/>
```

Comment:
This line should be `defer script/defer|nothing" ></script>` to fix #2870

Reply:
I missed that - yes that's necessary. See: https://github.com/plone/Products.CMFPlone/pull/2871